optimizely/project_config.py

```diff
@@ -41,12 +36,14 @@ def __init__(self, datafile, logger, error_handler):
     """
 
     config = json.loads(datafile)
-    self.parsing_succeeded = False
     self.logger = logger
     self.error_handler = error_handler
     self.version = config.get('version')
-    if self.version in UNSUPPORTED_VERSIONS:
-      return
+    if self.version not in SUPPORTED_VERSIONS:
```

Comment:
can't u just move this into method?

Reply:
I think this is cleaner and easier to see. This will not be reused anyway.